components/usage/cmd/run.go

```diff
@@ -74,7 +78,8 @@ func run() *cobra.Command {
 
 	cmd.Flags().BoolVar(&verbose, "verbose", false, "Toggle verbose logging (debug level)")
 	cmd.Flags().DurationVar(&schedule, "schedule", 1*time.Hour, "The schedule on which the reconciler should run")
-	cmd.Flags().StringVar(&apiKeyFile, "api-key-file", "/stripe-secret/apikeys", "Location of the stripe credentials file on disk")
+	cmd.Flags().BoolVar(&paymentEnabled, "enable-payment", false, "Whether or not payment integration with Stripe should be enabled")
+	cmd.Flags().StringVar(&apiKeyFile, "api-key-file", "/stripe-secret/apikeys", "Location of the Stripe credentials file on disk")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&apiKeyFile, "stripe-secret-path", "/stripe-secret/apikeys", "Location of the Stripe credentials file on disk")
```

Reply:
There may be other API keys in the future.

Reply:
Which other keys do you forsee we'll need here? Can we make this change when if and when we need to?

Reply:
It's more the point of being more specific, when we can. `api-key-file` doesn't tell me which API it is for, while `stripe-something-something` does